src/jit/compiler.h

```diff
@@ -7649,10 +7649,10 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     static AssemblyNamesList2* s_pAltJitExcludeAssembliesList;
 #endif // ALT_JIT
 
-#ifdef DEBUG
-
     static bool s_dspMemStats; // Display per-phase memory statistics for every function
```

Comment:
It most likely is - I'll wrap the thing (and the one below) within a conditional block and see if anything breaks. Thanks for catching this - I was trying to make the least number of changes but in this case it makes sense to add more PP directives.